core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -37,24 +43,33 @@ oppia.constant('HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS', {
 oppia.controller('Dashboard', [
   '$scope', '$rootScope', '$window', 'oppiaDatetimeFormatter', 'alertsService',
   'DashboardBackendApiService', 'RatingComputationService',
-  'ExplorationCreationService', 'FATAL_ERROR_CODES', 'UrlInterpolationService',
-  'EXPLORATIONS_SORT_BY_KEYS', 'HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS',
+  'ExplorationCreationService', 'UrlInterpolationService', 'FATAL_ERROR_CODES',
+  'EXPLORATION_DROPDOWN_STATS', 'EXPLORATIONS_SORT_BY_KEYS',
+  'HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS',
   function(
       $scope, $rootScope, $window, oppiaDatetimeFormatter, alertsService,
       DashboardBackendApiService, RatingComputationService,
-      ExplorationCreationService, FATAL_ERROR_CODES, UrlInterpolationService,
-      EXPLORATIONS_SORT_BY_KEYS, HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS) {
+      ExplorationCreationService, UrlInterpolationService, FATAL_ERROR_CODES,
+      EXPLORATION_DROPDOWN_STATS, EXPLORATIONS_SORT_BY_KEYS,
+      HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS) {
     var EXP_PUBLISH_TEXTS = {
       defaultText: (
         'This exploration is private. Publish it to receive statistics.'),
       smText: 'Publish the exploration to receive statistics.'
     };
 
+    $scope.activeExplorationId = '';
+    // Keeps track of the sub-dropdown that is opened in the main exploration
+    // dropdown.
+    $scope.statsDropdownOpened = {};
```

Comment:
I have the same question about this -- This could, like `activeExplorationId` above, just be a string, couldn't it? At most one stats dropdown is opened.

Unless the intended behavior is that if I switch between the dropdowns of 2 explorations, which sub-dropdown is opened is preserved? I think this is not necessarily, however.